pages/dkp/kaptain/2.0.0/install/requirements/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Requirements
+title: Requirements
+menuWeight: 3
+excerpt: Requirement reference values
+beta: false
+enterprise: false
+---
+
+The amount of resources required for your project **depends on the workload** you want to run, you need to consider: 
+
+-   The number, complexity, and size of your training jobs, notebook servers, deployed models, etc.
+-   The amount of metadata and log data stored with each run.
+
+To estimate the amount of resources required for your project, it is recommended that you test your specific use case.
```

Comment:
@bruced2iq, @dmennell is there anything else here we could mention to help customers? How do customers currently estimate their resources? Maybe we could include 1 or 2 lines on that here?